cpp/include/cuspatial/shp_readers.hpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+typedef struct gdf_column_ gdf_column; // forward declaration
+
+namespace cuspatial {
+
+/**
+ * @brief read polygon data from file in SoA format
```

Comment:
I don't think SoA format is a standard terminology in GIS, is it? If not, let's not use jargon if we can avoid it. 

```suggestion
 * @brief read polygon data from an ESRI Shapefile.
```

Reply:
this is a typo - thanks for the correction.